Apps/APN-UIKit/APN UIKit/APN UIKit/View/LoginViewController.swift

```diff
@@ -49,14 +50,19 @@ class LoginViewController: UIViewController {
             showAlert(withMessage: "Please fill all fields", .error)
             return
         }
+        guard let emailId = emailTextField.text, let name = firstNameTextField.text else {
+            return
+        }
+        CustomerIO.shared.identify(identifier: emailId, body: ["firstName" : name])
+        storage.userEmailId = emailId
+        storage.userName = name
```

Comment:
What is the reason we are storing the email and username in the app? Does this provide value to the QA team?

Reply:
Yes, I am displaying username and email Id in the dashboard. Figma UI does not contain any user detail hence it becomes hard for QA and dev team as well to remember the login details, specially in the case of a guest login.